src/v0/destinations/lemnisk/utils.js

```diff
@@ -0,0 +1,33 @@
+const { TransformationError } = require("../../util/errorTypes");
+/**
+ * @param {*} message input message
+ * @returns Page Event name with category
+ */
+const generatePageName = message => {
+  const { name, category } = message;
+  const pageCat = category ? "".concat(category, " ") : "";
+  const pageName = name ? "".concat(name, " ") : "";
+  const generatedName = `${pageCat}${pageName}`;
+  const validName =
+    generatedName.length > 20 ? generatedName.substring(0, 19) : generatedName;
+  const eventName = `Viewed ${validName} Page`;
+  return eventName;
+};
```

Comment:
Do they have implemented this logic? I am not seeing this in their documentation.